### PR TITLE
Changes “back” button appearance; adjusts spacing and layout

### LINKED ISSUE #290: Update button treatment/spacing re: discussion w/Sebastian
- [x] change “back” button appearance to match death’s (i.e. text link)
- [x] increase vertical spacing between question options and the back/next buttons
- [x] yes/no/i don’t know labels should move next to icons on mobile vs. underneath at wider viewports

### PR DESCRIPTION
**Pull request by jessicamarcus**
Visual changes to the question flow as recently discussed; closes #290.

### CODE REVIEW

File: services-js/registry-certs/client/birth/styling.tsx [hunk 1]
```diff
@@ -20,6 +20,18 @@ const FOCUS_STYLE = {
   outlineOffset: 1,
 };
 
+export const BACK_BUTTON_STYLING = css({
```
Comment (fionawhim) [inline services-js/registry-certs/client/birth/styling.tsx]: You should be able to use the `lnk` class on a `button` to achieve this.

File: services-js/registry-certs/client/birth/questions/QuestionComponent.tsx [hunk 2]
```diff
@@ -24,12 +24,12 @@ export default function QuestionComponent(props: Props): JSX.Element {
...
 
       <div className="g g--r m-t700">
-        <div className="g--6 ta-r m-b500">
+        <div className="g--6 ta-r m-v500">
           {props.handleProceed &&
             !props.startOver && (
               <button
                 type="button"
-                className="btn btn--br btn--b-sm"
+                className="btn btn--b-sm"
```
Comment (fionawhim) [inline services-js/registry-certs/client/birth/questions/QuestionComponent.tsx:34]: Per Sebastian, we should switch away from the bordered Next buttons all across the app.

Can you switch the checkout flow back buttons to links as well?

**Review by fionawhim — COMMENTED** [review]
Overall the radio button changes are looking really good.

The vertical alignment on "how are you related" is a little off, though (at 320): https://percy.io/CityOfBoston/Registry-Certs/builds/1402841/view/92397119/320?mode=diff&browser=firefox&snapshot=92397119

Also, the multiple items radio isn't wrapping: https://percy.io/CityOfBoston/Registry-Certs/builds/1402841/view/92397136/320?mode=diff&browser=firefox&snapshot=92397136

File: services-js/registry-certs/client/birth/questions/QuestionComponent.tsx (hunk 2 above)
Comment (jessicamarcus) [reply services-js/registry-certs/client/birth/questions/QuestionComponent.tsx:34]: Thanks for the catch!

File: services-js/registry-certs/client/birth/styling.tsx (hunk 1 above)
Comment (jessicamarcus) [reply services-js/registry-certs/client/birth/styling.tsx]: Ah, thanks! I _thought_ I recalled a class for that, but didn’t track it down when I searched through the fleet stylesheets.

File: services-js/registry-certs/client/birth/ReviewRequestPage.tsx [hunk 3]
```diff
@@ -139,23 +141,13 @@ export default class ReviewRequestPage extends React.Component<Props> {
 
         <div className="m-t700 g g--r">
           <div className="g--6 ta-r m-b500">
-            <button
-              className="btn btn--b-sm btn--br"
-              type="button"
-              onClick={this.goToCheckout}
-            >
+            <button className="btn" type="button" onClick={this.goToCheckout}>
```
Comment (fionawhim) [inline services-js/registry-certs/client/birth/ReviewRequestPage.tsx]: This should keep the `btn--b-sm` class

File: services-js/registry-certs/client/birth/components/BackButton.tsx [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+
+interface Props {
+  handleClick: () => void;
+}
+
+/**
+ * “Back” button that looks like a text link. Container div provides padding
+ * to match that of the “next” button.
+ */
+export default function BackButton(props: Props): JSX.Element {
```
Comment (fionawhim) [inline services-js/registry-certs/client/birth/components/BackButton.tsx:13]: 👍  for factoring this out.

File: services-js/registry-certs/client/birth/components/QuestionComponent.tsx [hunk 5]
```diff
@@ -42,7 +44,7 @@ export default function QuestionComponent(props: Props): JSX.Element {
             props.startOver && (
               <button
                 type="button"
-                className="btn btn--b-sm btn--br btn--w"
+                className="btn btn--w"
```
Comment (fionawhim) [inline services-js/registry-certs/client/birth/components/QuestionComponent.tsx]: think you want to keep `btn--b-sm` but not `btn--w`

**Review by fionawhim — APPROVED** [review]

**Pull request merged**